flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/nodes/exec/serde/JsonSerdeUtil.java

```diff
@@ -103,8 +103,14 @@ public static boolean hasJsonCreatorAnnotation(Class<?> clazz) {
         OBJECT_MAPPER_INSTANCE.registerModule(createFlinkTableJacksonModule());
     }
 
-    /** Get the {@link ObjectMapper} instance. */
-    public static ObjectMapper getObjectMapper() {
+    /**
+     * Get the {@link ObjectMapper} instance.
+     *
+     * <p>This is not exposed outside this package to avoid bad usages, like adding new modules. If
+     * you need to read and write json, just use {@link #createObjectWriter(SerdeContext)} and
+     * {@link #createObjectReader(SerdeContext)}.
+     */
+    static ObjectMapper getObjectMapper() {
```

Comment:
Do we actually need to expose it? All usages can either use the `ObjectMapper` of `JsonSerdeTestUtil` or are SerdeTests that need an update anyway to `toJson/toObject`.

Reply:
I removed the method completely, check the last commit